imports/plugins/included/taxes-avalara/server/methods/avalogger.js

```diff
@@ -0,0 +1,42 @@
+import bunyan from "bunyan";
+import { Logs } from "/lib/collections";
+import { Reaction } from "/server/api";
+
+const level = "INFO";
+
+class BunyanMongo {}
+
+
+BunyanMongo.prototype.write = Meteor.bindEnvironment((logData) => {
```

Comment:
This seems weird to have a blank class.

Maybe it could be written as:
```
class BunyanMongo {
   write = Meteor.bindEnviroment((logData) => {

   })
}
```

**Doesn't need to be changed, just another option for the future.**